src/name_core.jl

```diff
@@ -110,22 +138,16 @@ function remaining_dimnames_after_dropping(dimnames::Tuple, dropped_dim::Integer
 end
 
 function remaining_dimnames_after_dropping(dimnames::Tuple, dropped_dims)
-    # Note: This allocates once, and it shouldn't have to. Reason is same as for
-    # remaining_dimnames_from_indexing. I.e. returning tuple of symbols allocates.
-    # see `@btime remaining_dims_names_from_reducing((:a,:b, :c, :d, :e), (1,2,))
+    # 1-Allcoation see: `@btime remaining_dimnames_after_dropping((:a,:b,:c,:d,:e), (1,2,))
 
 
     anti_names = identity_namedtuple(map(x->dimnames[x], dropped_dims))
     full_names = identity_namedtuple(dimnames)
 
     # Now we construct a new named tuple, with all the names we want to remove at the start
-    combined_names = merge(anti_names, full_names)
+    combine_names = merge(anti_names, full_names)
     n_skip = length(anti_names)
     ntuple(length(full_names) - n_skip) do ii
-        combined_names[ii + n_skip]  # Skip over the ones we left as the start
+        combine_names[ii + n_skip]  # Skip over the ones we left as the start
```

Comment:
```suggestion
        combined_names[ii + n_skip]  # Skip over the ones we left as the start
```
else confusing with the function of the same name (also needs changing a few lines above)